src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+
+type Props = {
+  people: Person[] | null;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personId } = useParams();
+  const [activePersonName, setActivePersonName] = useState<string | null>(null);
+
+  const isPersonInTable = (personName: string) => {
+    return people?.find(({ name }) => name === personName);
+  };
+
+  const handleLinkForParent = (personName: string) => {
+    if (!people) {
+      return;
+    }
+
+    if (isPersonInTable(personName)) {
+      const indexOfPerson = people?.findIndex(
+        person => person.name === personName,
+      );
+      const person = people[indexOfPerson];
```

Comment:
The variable `indexOfPerson` is used to find the person object, but it could be simplified by directly using `find` to get the person object. This would make the code cleaner and avoid unnecessary indexing.